### PR TITLE
Remove code conditional on mpp_luncheck.is_RdacLun

### PR DESCRIPTION
**Pull request by MarkSymsCtx**
This has only ever returned False for some time

Signed-off-by: Mark Syms <mark.syms@citrix.com>

### CODE REVIEW

**Comment by MarkSymsCtx** [comment]
Still testing this.

File: mk/sm.spec.in [hunk 1]
```diff
@@ -9,18 +9,20 @@ License: LGPL
 URL:  http://www.citrix.com
 Source0: sm-@SM_VERSION@.tar.bz2
 BuildRoot: %{_tmppath}/%{name}-%{version}-root
-BuildRequires: python-devel xen-devel systemd
+BuildRequires: python-devel xen-devel systemd pylint python-nose python-coverage python2-mock python2-bitarray
```
Comment (chandrikas) [inline mk/sm.spec.in]: Unrelated changes?

File: mk/sm.spec.in [hunk 2]
```diff
@@ -36,19 +38,20 @@ rm -rf $RPM_BUILD_ROOT
 [ ! -x /sbin/chkconfig ] || chkconfig --add sm-multipath
 service sm-multipath start
 %systemd_post make-dummy-sr.service
+%systemd_post mpcount.service
 %systemd_post snapwatchd.service
-%systemd_post updatempppathd.service
-%systemd_post usb-scan.service
+%systemd_post xs-sm.service
```
Comment (chandrikas) [inline mk/sm.spec.in]: Unrelated changed?

**Comment by robertbreker** [comment]
Random thought: How does this change interfere with customers, who have LUNs with MPP-RDAC parameters on them?

(I don't know the answer to this. But I'd expect we at least don't support direct upgrades from any releases that still use MPP-RDAC.)

File: mk/sm.spec.in (hunk 2 above)
Comment (MarkSymsCtx) [reply mk/sm.spec.in]: sort of, synchronising the spec with the one being used for product

**Comment by MarkSymsCtx** [comment]
It hasn't semantically changed the code at all. All the code removed used to call `mpp_luncheck.is_RdacLun` which has since `ba133844 (Chandrika Srinivasan 2014-06-24` returned False under all conditions.

File: mk/sm.spec.in (hunk 2 above)
Comment (MarkSymsCtx) [reply mk/sm.spec.in]: As the spec needed changing in order to remove the files and services anyway.

**Comment by chandrikas** [comment]
Yes, support for RDAC LUN was deprecated a long time ago

File: drivers/mpath_dmp.py [hunk 3]
```diff
@@ -200,10 +174,7 @@ def refresh(sid,npaths):
             scsiutil.rescan(scsiutil._genHostList(""))
             if not util.wait_for_path(path,60):
                 raise xs_errors.XenError('Device not appeared yet')
-        if not (mpp_luncheck.is_RdacLun(sid)):
-            _refresh_DMP(sid,npaths)
-        else:
-            _refresh_MPP(sid,npaths)
```
Comment (chandrikas) [inline drivers/mpath_dmp.py]: Remove definition for _refresh_MPP too

**Comment by chandrikas** [comment]
Good to be merged
